### PR TITLE
Add logging to opa-connector

### LINKED ISSUE #1472: Add logging to opa-connector
Currently, there is no logging done to the log which makes it hard to debug a fail.

### PR DESCRIPTION
**Pull request by revit13**
Closes #1472

### CODE REVIEW

File: connectors/opa/connector.go [hunk 1]
```diff
@@ -28,13 +30,15 @@ type ConnectorController struct {
 	OpaServerURL  string
 	OpaClient     *retryablehttp.Client
 	CatalogClient clients.DataCatalog
+	log           zerolog.Logger
```
Comment (shlomitk1) [inline connectors/opa/connector.go:33]: Make it with capital letter for consistency

**Comment by shlomitk1** [comment]
How does it solve the issue? Where is the actual logging?

**Comment by revit13** [comment]
@shlomitk1 will add the logging next

**Comment by revit13** [comment]
@shlomitk1 will add it next. Thanks

**Review by shlomitk1 — APPROVED** [review]

**Pull request merged**